package.json

```diff
@@ -10,7 +10,7 @@
     "coverage": "c8 --reporter=html --reporter=lcov npm run test",
     "build": "tsc",
     "protobuf": "protocol-buffers feeditem.proto -o src/feeditem.js",
-    "postinstall": "husky install"
+    "postinstall": "husky-run install"
```

Comment:
Where did this break?

Reply:
Not sure, I only noticed after I did an install of hyper-sdk.